### ISSUE TITLE
The result is 0.

### ISSUE THREAD

**Issue by lalala1020** [Anery/RSAN#11]
Excuse me，when training the model, all the indicators in the validation set test have specific values, but after the training, all the indicators in the test set test are 0.what is the reason? Thank you！

**Comment by xspring14** [comment]
me to
